releasenotes/notes/fix-keep-copyreg-29c28bcc854e13bb.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fix a bug that caused traceback objects to fail to pickle when using gevent.
```

```suggestion
    gevent: Fix a bug that caused traceback objects to fail to pickle when using gevent.
```
? I guess it's kind of repetitive, but I like it when the fixes get categorized a bit.